sdk/nodejs/resource.ts

```diff
@@ -280,6 +280,11 @@ export class ComponentResource extends Resource {
 (<any>ComponentResource).doNotCapture = true;
 (<any>ComponentResource.prototype).registerOutputs.doNotCapture = true;
 
+/* @internal */
+export const testingOptions = {
+    isDryRun: false,
+};
```

Comment:
This is a little unfortunate - if we are going to do something like this - it would be great if there were a way to control this at the `runtime.isDryrun` level overall to ensure some consistency around the state of this (and provide a general way to test dry runs in the SDK). Any way to do that instead?

Reply:
i'll look into this.